### ISSUE TITLE
Feature request: when a kdoc consists only of @see, copy it.

### ISSUE THREAD

**Issue by mikehearn** [Kotlin/dokka#202]
A KDoc comment like `/** see [bar] */` could simply use the docs for bar for this specific item. If the thing being documented is also called [bar] it could find the first overload that has non-`@see` docs and use that. This would make documenting methods that are minor variants of other methods easier.

**Comment by Xerus2000** [comment]
YES! I would love this! Often you create util methods that just bridge to another original method.
